packages/rancher-vsphere-csi/charts/questions.yaml

```diff
@@ -44,7 +44,20 @@ questions:
     description: Enable migration of volumes provisioned by in-tree vSphere provider to CSI (Available for vSphere 7.0 U1+ only)
     type: boolean
     default: false
-    group: Migration
+    group: "Features"
+
+  - variable: csiAuthCheck.enabled
+    label: Enable authorization check on datastores and get datastores
```

Comment:
Not sure I understand why this label means. Do you think a better phrasing of this would be:

`Enable authorization checks on operations involving datastores`

Reply:
I couldn't find a good description in their docs stating what the csi-auth-check is exactly, so I used [this](https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/73f89c21a72a8921a0458b2e34a6f5e0e3179c43/pkg/csi/service/common/authmanager.go#L39-L40) comment since this option enables that service. I like your suggestion better though, so I'll update it.

Reply:
updated